src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -1,6 +1,8 @@
 package core.mate.academy.model;
 
-public abstract class Machine implements Workable {
+import core.mate.academy.service.MachineProducer;
+
+public abstract class Machine implements Workable, MachineProducer {
```

Comment:
```suggestion
public abstract class Machine implements Workable {
```
you should create 3 Producer classes (1 for each type of machine) and override `get()` method there